daml-lf/repl/src/main/scala/com/digitalasset/daml/lf/repl/testing/Main.scala

```diff
@@ -611,6 +626,58 @@ object Repl {
       .getOrElse((false, state))
   }
 
+  val warmingTime = 20L
+
+  private[this] def standardDeviation(a: Seq[Long]): Double = {
+    def avg = a.map(_.toDouble).sum / a.size
+    math.sqrt(a.map(x => (x - avg) * (x - avg)).sum / a.size)
+  }
+
+  def cmdBench(state: State, testId: String): (Boolean, State) = {
```

Comment:
I think I feel slightly uneasy about reinventing JMH here. Is there a way we can just delegate to that?